docs/app-dev/indexing-transactions.md

```diff
@@ -177,5 +233,44 @@ You can query for a paginated set of blocks by their events by calling the
 curl "localhost:26657/block_search?query=\"block.height > 10 AND val_set.num_changed > 0\""
 ```
 
+## `match.events` keyword 
```

Comment:
Given that this is automatically added by way of the RPC flag, and we'll be changing the default behaviour to effectively have `match.events = 1` from Tendermint v0.37 onwards, should we be advertising this keyword in the docs?

It does, however, make sense to me to document the `match_events` RPC parameter though - please add this to the [OpenAPI spec](https://github.com/tendermint/tendermint/blob/v0.34.x/rpc/openapi/openapi.yaml)?

Reply:
I agree, I'll edit all the docs to not mention the match.events=1 keyword, rather just match the RPC.